internal/lpastore/client.go

```diff
@@ -214,6 +214,68 @@ func (c *Client) SendLpa(ctx context.Context, donor *actor.DonorProvidedDetails)
 	if err != nil {
 		return err
 	}
+
+	return c.do(ctx, req)
+}
+
+type updateRequest struct {
+	Type    string                `json:"type"`
+	Changes []updateRequestChange `json:"changes"`
+}
+
+type updateRequestChange struct {
+	Key string `json:"key"`
+	Old any    `json:"old,omitempty"`
```

Comment:
We don't appear to be using this (yet) but the [API spec](https://github.com/ministryofjustice/opg-data-lpa-store/blob/main/docs/openapi/openapi.yaml#L454) states it is a required field. Is this in anticipation of a future requirement?